.github/workflows/frontend-ci.yml

```
@@ -30,5 +32,33 @@ jobs:
     - name: Install dependencies
       run: pnpm install --frozen-lockfile
 
+    - name: Switch platformAdapter to Web adapter
```

```suggestion
    - name: Switch platformAdapter to Web adapter
       shell: bash
```

Setting the shell to `bash` would probably work for the [Windows issue](https://github.com/infinilabs/coco-app/actions/runs/18522233606/job/52784615458?pr=927)